solarforecastarbiter/reports/templates/pdf/macros.j2

```diff
@@ -115,7 +115,7 @@
     \bottomrule
     \endlastfoot
     %- for pfxobs in report.raw_report.processed_forecasts_observations
-    \VAR{pfxobs.name} &
+    \VAR{pfxobs.name | replace("<=", "$\leq$") | replace("%", "\%")} &
```

Comment:
why not use the html_to_tex filter?

Reply:
ah, I did not understand until now that it's made accessible for calling within a jinja variable expression.